src/routes/DebtPositionDetail/DebtPositionDetail.test.tsx

```diff
@@ -0,0 +1,238 @@
+import { screen, fireEvent } from '@testing-library/react';
+import DebtPositionDetail from './DebtPositionDetail';
+import { describe, it, expect, beforeEach, vi, afterEach } from 'vitest';
+import { i18nTestSetup } from '../../__tests__/i18nTestSetup';
+import { render } from '../../__tests__/renderers';
+import { mockData } from './mocks/apiResponse';
```

Comment:
We could also try to generate a mock using zodock instead of creating the mock manually 
```typescript
const mockSchema = createMock(schema);
```